### ISSUE TITLE
Generate 500 first, continue generating 9500 later

### ISSUE THREAD

**Issue by kbb0000** [HashLips/hashlips_art_engine#760]
Hi I am thinking about generating 500 pieces from my layers. Then I want to add new layers in 2 weeks for more variation and generate the remaining 9500. Is this possible without same dna or any kind of issues like that?

**Comment by bolshoytoster** [comment]
@kbb0000 it's possible in [my repo](https://github.com/bolshoytoster/hashlips_art_engine/). You can just set `growEditionSizeTo` to `1000` and it will generate the remaining ones.

**Comment by kbb0000** [comment]
> @kbb0000 it's possible in [my repo](https://github.com/bolshoytoster/hashlips_art_engine/). You can just set `growEditionSizeTo` to `1000` and it will generate the remaining ones.

cool thanks!

**Comment by Cyncra** [comment]
> Hi I am thinking about generating 500 pieces from my layers. Then I want to add new layers in 2 weeks for more variation and generate the remaining 9500. Is this possible without same dna or any kind of issues like that?

can u add your continuing generate function to this repo?

https://github.com/nftchef/art-engine

**Comment by Cyncra** [comment]
> @kbb0000 it's possible in [my repo](https://github.com/bolshoytoster/hashlips_art_engine/). You can just set `growEditionSizeTo` to `1000` and it will generate the remaining ones.

@bolshoytoster 

can u add your continuing generate function to this repo?

https://github.com/nftchef/art-engine

**Comment by bolshoytoster** [comment]
@Cyncra I could try to.

**Comment by ultraviolet007rainbow** [comment]
You mean you set groweditionsizeto to 10000 not 1000 in this case yes?

**Comment by bolshoytoster** [comment]
@ultraviolet007rainbow yes sorry

**Comment by ultraviolet007rainbow** [comment]
I tried to continue from a batch of 4500 that stopped. So I was hoping to restart it, but when I tried running your version of the engine, to finish out the 10000 which would be randomly numbered as the first 4500 were. It started giving me edition numbers larger than 10000, like 20606, 17300, 12190, etc. It also reused 2 numbers that were already used previously. Not sure what happened, is there some kind of incompatibility with previously generated batches that use the 
const shuffleLayerConfigurations = true; instead of false? It would be a great thing to be able to pickup where you left off if the system crashed, you had to stop it, there was an error, or whatever. I copied the build folder with all the original JSON and image files over, but there was never a _metadata.json generated, only the individual JSON files for each image, not sure why, other than the original batch stopped after the first 4500 because there was an mismatched folder name in the next 4500 layer description. Does it only generate that master JSON at the end of the 10000?

**Comment by Cyncra** [comment]
> @Cyncra I could try to.
I am very pleased with your reply.

It's a function I've been looking for, and I hope it works like yours in that repo.

**Comment by bolshoytoster** [comment]
@ultraviolet007rainbow
> but there was never a _metadata.json generated

That'll be the issue, this uses the `_metadata.json` to load the dna. I'll update it to use the individual files.

**Comment by kbb0000** [comment]
I tried @bolshoytoster's repo and it works

**Comment by ultraviolet007rainbow** [comment]
> @ultraviolet007rainbow
> 
> > but there was never a _metadata.json generated
> 
> That'll be the issue, this uses the `_metadata.json` to load the dna. I'll update it to use the individual files.

Yes, I figured it probably had something to do with that, thanks :)

**Comment by Cyncra** [comment]
@bolshoytoster 

awsome.
Can you take a look at my request?
https://github.com/nftchef/art-engine

**Comment by bolshoytoster** [comment]
@Cyncra I'm working on it now :).

**Comment by Cyncra** [comment]
@bolshoytoster 

Thanks a lot!!

**Comment by bolshoytoster** [comment]
@Cyncra I don't think I can do it with nftchef's repo because of the way they handle dna. You can ask on that repo linking [this](https://github.com/bolshoytoster/hashlips_art_engine/commit/d8541a16808c2beec18812344d3bb732c7c7e3c8#diff-27653c212e1cfe533e4eb2f7d0d3f89604c9de48a09583b4cbbbcbd08a07da79) commit if they want to add it.
